src/windows/ms2mit/ms2mit.c

```diff
@@ -39,20 +41,61 @@ xusage(void)
     exit(1);
 }
 
-void
-main(
-    int argc,
-    char *argv[]
-    )
+/* Return true if princ is not a cross-realm principal. */
```

Comment:
... and is a TGT principal?  I suggest "Return true if princ is a local (not cross-realm) krbtgt principal."